tests/recipe_tests/test_XarrayZarrRecipe.py

```diff
@@ -340,6 +345,19 @@ def test_chunks_distributed_locking(
     )
 
 
+def test_no_consolidate_dimension_coordinates(netCDFtoZarr_recipe):
+    RecipeClass, file_pattern, kwargs, ds_expected, target = netCDFtoZarr_recipe
+
+    rec = RecipeClass(file_pattern, **kwargs)
+    rec.consolidate_dimension_coordinates = False
```

Comment:
Perfect...my one last comment was going to be that we needed a test for both options, but that's already here.